api/worker_grpc.go

```diff
@@ -686,6 +686,65 @@ func notifyWorkersPremieres(daoWrapper dao.DaoWrapper) {
 	}
 }
 
+type generateVideoSectionImagesParameters struct {
+	sections                                    []model.VideoSection
+	playlistUrl, courseName, courseTeachingTerm string
+	courseYear                                  uint32
+}
+
+func GenerateVideoSectionImages(daoWrapper dao.DaoWrapper, parameters *generateVideoSectionImagesParameters) error {
+	workers := daoWrapper.WorkerDao.GetAliveWorkers()
+	workerIndex := getWorkerWithLeastWorkload(workers)
+	workers[workerIndex].Workload += 1 // Not sure if that's a reasonable amount.
```

Comment:
No need to modify this variable, the worker reports its own workload that is set in its `status` package, also this is such a low workload, we don't need to increase it here imo.